recipes/nsearch/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+cmake -Wno-dev ..
+cmake --build . --config Release
```

Comment:
Is there a make check/test or ctest we can run here?

Reply:
`make check` can be run there.
I'll add that as a commit.